resolwe_bio/processes/workflows/wgbs.yml

```diff
@@ -68,30 +140,24 @@
             Merging CpG pairs results in symmetric methylation levels. Methylation is usually
             symmetric (cytosines at CpG sites were methylated on both DNA strands). Choosing this
             option will only keep the CpG sites data.
+    - name: markduplicates
```

Comment:
We also have a `rm_dup` parameter in the alignment section. Couldn't we remove the whole MarkDuplicates parameters section and set the correct inputs to the MarkDuplicates based on what was selected in `rm_dup`? This only applies if a "correct" way of setting those parameters together exists, e.g. if one was set to False, the other should be True. This would significantly simplify the inputs for all users, since its not clear why we are dealing with remove_duplicates-related inputs in two sections.

Reply:
I completely agree this just complicates the whole input and should be simplified.